### ISSUE TITLE
hope the model will be updated to the coreML version of Apple silicon (someone provides a ready-made model)

### ISSUE THREAD

**Issue by ghost** [Vaida12345/waifuExtension#18]
https://github.com/john-rocky/CoreML-Models
This repository provides a coreML version of almost all models, hoping to produce a separate application version of waifuExtension for Apple silicon to improve efficiency by using the coreML models.

**Comment by Vaida12345** [comment]
Yes, I have realized this a long time ago, however, the translated coreML models requires some additional work that I am not capable of dealing right now, such as scaling alpha channels. Moreover, to my disappointment, CoreML models are not significantly faster than the GPU implementations.

**Comment by ghost** [comment]
Using the Diffusion model of coreML to generate AI art in large quantities on M1 Max will hardly generate heat, and fans will not start very often. The advantages of coreML may be more reflected in low power consumption. In the same way, it may be true that using coreML for image high-resolution models can not further improve the speed, but it may have better performance in power consumption and heat generation, which is just my guess.
